### DISCUSSION TITLE
去中心化的安全性

### DISCUSSION THREAD

**Discussion by yuhao112** [1101-GS4719-C/Python-Community-GS4719-C#548] [科技新知]
https://support.opensea.io/hc/zh-tw/articles/4408474998419-%E5%A6%82%E4%BD%95%E5%9C%A8NFT%E7%9A%84%E4%B8%96%E7%95%8C%E8%A3%A1%E7%A2%BA%E4%BF%9D%E5%AE%89%E5%85%A8-
![image](https://user-images.githubusercontent.com/100193736/162381147-2954fadf-4a68-4d6e-8d2b-a1fafd
在愚人節當天，周杰倫也發出了聲明說他的NFT被盜了，且並非是愚人節的玩笑。如此詐騙的案例層出不窮，網路上查詢方法，也只會告訴你買個冷錢包以策安全，然後私鑰不要截圖放在手機照片裡面。目前看到最多都是這樣，雖然說區塊鏈說明每筆交易都是公開透明的，但偏偏在被盜取後，就算追蹤到錢包，NFT也會立即被轉手，因此連追回來都是不太可能的，項目方也都不存在任何補償機制。對我來說，依舊是認為區塊鏈技術上安全的漏洞真是太大了，或許這就是去中心化帶來的風險，縱使他帶給我們無比的自由。

**Comment by treeann** [comment]
NFT真的是一門學問

**Comment by PoooYen** [comment]
虛擬的東西在發展初期存在著巨大的風險...

**Comment by etin2468** [comment]
自由的代價就是安全性問題啊...

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，安全性有待加強。

**Comment by HsuChen88** [comment]
原以為去中心化的東西才是分散風險的保障，但顯然並非如此。

**Comment by handsomeniko** [comment]
資安問題是蠻重要的議題

**Comment by Sazmafuji** [comment]
NFT剛開始，期待NFT的發展
